extensions/sendgrid-extension/v1/actions/sendEmailWithTemplate/config/fields.ts

```diff
@@ -93,5 +94,5 @@ export const FieldsValidationSchema = z.object({
       }
     }),
   fromName: z.string().optional(),
-  fromEmail: z.string().email().optional(),
+  fromEmail: getEmailValidation().optional(),
```

Comment:
can we chain .optional() with a custom function?

Reply:
Not sure if I got your question. We can use optional because the function returns a`ZodTypeAny`